nipoppy_cli/nipoppy/workflows/base.py

```diff
@@ -35,6 +35,10 @@ class BaseWorkflow(Base, ABC):
     log_prefix_run_stderr = "[RUN STDERR]"
     validate_layout = True
 
+    # hack to avoid errors when loading/processing the default config
```

Comment:
For workflow objects that don't have `pipeline_name` and/or `pipeline_version` init arguments (`DicomReorgWorkflow` and `DoughnutWorkflow`), loading the config will fail because it will try to do the string replacement in the pipeline config, but there is no pipeline name/version to replace them.

Eventually if we come up with a better way to do this replacement thing, it might be worth implementing so that not all pipeline name/version placeholders in the config get replaced by the same thing...

Reply:
I see - always messy to deal with mix of standard and non-standard workflows :-/